packages/graphback-runtime-knex/src/createKnexServices.ts

```diff
@@ -6,38 +6,18 @@ import { PgKnexDBDataProvider } from './PgKnexDBDataProvider';
 import { KnexDBDataProvider } from './KnexDBDataProvider';
 
 /**
- * Helper function for creating array of datasources based on the model files that will 
+ * Helper function for creating array of datasources based on the model files that will
  * be able to connect to Postgres database using knex
- * 
- * @param schema 
- * @param db 
- * @param pubSub 
+ *
+ * @param schema
+ * @param db
+ * @param pubSub
  */
 export const createKnexPGCRUDRuntimeServices = (
   models: GraphbackPubSubModel[], schema: GraphQLSchema,
   db: Knex, pubSub: PubSubEngine
 ) => {
-  if (!models || models.length === 0) {
-    throw new Error(`No models provided`)
-  }
-
-  return models.reduce((services: any, model: GraphbackPubSubModel) => {
-    const modelType = schema.getType(model.name) as GraphQLObjectType
-    if (modelType === undefined) {
-      throw new Error(`
-      Schema is missing provided type.
-      Please make sure that you pass the right schema to createCRUDRuntimeContext`)
-    }
-
-    const objectDB = new PgKnexDBDataProvider(modelType, db)
-
-    services[model.name] = new CRUDService(modelType, objectDB, {
-      pubSub,
-      ...model.pubSub
-    })
-
-    return services;
-  }, {})
+  throw new Error("Deprecated")
```

Comment:
Like that we actually giving this backwards compatibility. We can even return link to docs :D